plugins/aws/s3/src/hopeit/aws/s3/object_storage.py

```diff
@@ -199,7 +200,10 @@ async def connect(
                 try:
                     await object_storage.create_bucket(Bucket=self.bucket, **kwargs)
                 except ClientError as e:
-                    if e.response["Error"]["Code"] == "BucketAlreadyOwnedByYou":
+                    if e.response["Error"]["Code"] in [
+                        "BucketAlreadyOwnedByYou",
+                        "BucketAlreadyExists",
```

Comment:
There might be a case were the bucket already exists but the aws user do not have permission to create buckets. Please address this a part of a ticket to make bucket creation optional.

Reply:
I have created https://github.com/hopeit-git/hopeit.aws/issues/6 to address this.